kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -290,6 +290,11 @@ form .input-group {
     background-color: #40775F !important;
 }
 
+.login-btn, btn-block {
+    display: block;
+    width: 100%;
```

Comment:
This is how it looks with simplified login:

![simplified-login-dialog](https://cloud.githubusercontent.com/assets/1457929/6988029/8e01350a-da4d-11e4-800f-cb846b6ba486.png)